lib/google/gax/api_callable.rb

```diff
@@ -132,8 +132,6 @@ def initialize(request_page_token_field,
       #   A proc to update the response object.
       # @param request [Object]
       #   The initial request object.
-      # @param settings [CallSettings]
-      #   The call settings to enumerate pages
       # @return [PagedEnumerable]
       #   returning self for further uses.
       def start(api_call, request, settings, block)
```

Comment:
Should `settings` still be in this method signature?

Reply:
Yeah, it might be a good idea to merge #166 before this PR.

Reply:
OK, I'll leave that up to you.

Reply:
This has been fixed.